### ISSUE TITLE
about the multi-part loss function

### ISSUE THREAD

**Issue by HHUZZK** [eriklindernoren/PyTorch-YOLOv3#175]
Hi:
I am just getting started with object dection and have some problems about the multi-part loss function. The loss function implemented in line 192 in models.py has some deviations from the original loss in the paper [https://arxiv.org/pdf/1506.02640.pdf](url). I take a screenshot of the formula shown below from the paper:
![image](https://user-images.githubusercontent.com/32475718/57745195-c4fa0080-76fe-11e9-9637-20f87df8471c.png)
The deviations can be listed as follows:
1. You use width and height directly instead of the square root of that.
2. You set λcoord=1 and λnoobj=100 instead of λcoord = 5 and λnoobj = 0.5.
i am grateful for any help :)

**Comment by developer0hye** [comment]
did you find the reason??

I also am curious about question 2...

did you train after changing that value??

**Comment by guuusss** [comment]
sorry,i did not.

**Comment by developer0hye** [comment]
I tested changing value on my dataset.

 self.obj_scale = 100
 self.noobj_scale = 1

But, I had a bad result.

**Comment by guuusss** [comment]
Thank you :)

**Comment by sydney0zq** [comment]
I am very confused about the question 2.

**Comment by ujsyehao** [comment]
you can refer to #295

**Comment by Flova** [comment]
I closed this issue due to inactivity. Feel free to reopen for further discussion.
